src/components/mermaidInputs/InputValidationInfo/InputValidationInfo.js

```diff
@@ -23,6 +23,7 @@ const InputValidationInfo = ({
   validationType,
   currentSelectValue,
   updateValueAndResetValidationForDuplicateWarning,
+  hasObservationRecords,
```

Comment:
I think this is too specific to a fish bin input. This component is a generic input component (generic to Mermaid anyway) Maybe the prop be more generic like `additionalText` and then below so `additionalText ? additionalText : null` and let the parent decide when to include the additional text. 

this way it is potentially resulable for other components that need non-validation helper text (but dont share the rule about observation tables)